src/contractWrappers/ArbitrableTransactionWrapper.js

```diff
@@ -195,13 +198,93 @@ class ArbitrableTransactionWrapper extends ContractWrapper {
         {
           from: account,
           gas: config.GAS,
-          value: 0
+          value: 0,
         }
       )
 
     return txHashObj.tx
   }
 
+  /**
+  * Call by partyA if partyB is timeout
+  * @param {string} account ETH address of user
+  * @param {string} contractAddress ETH address of contract
+  * @return {string} txHash Hash transaction
+   */
+  timeOutByPartyA = async (
```

Comment:
I think these names are a little confusing. It is unclear if timeout is being called on partyA or partyB. I would call these something like `callTimeOutPartyA`